fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -247,6 +254,11 @@ fun BottomSheet(
         tokens.scrimColor(bottomSheetInfo).copy(alpha = scrimOpacity)
 
     val scope = rememberCoroutineScope()
+    if(stickyBehavior){
+        if(stickyThresholdUpward == null || stickyThresholdDownward == null){
+            throw IllegalArgumentException("Sticky threshold should be provided for sticky behavior")
```

Comment:
we can have a default value set for both upward and downward drag instead of null and throwing exception

Reply:
Added def 56f